src/App.vue

```diff
@@ -128,9 +128,11 @@ export default {
     document.addEventListener('fullscreenchange', () => {
       if (document.fullscreenElement) {
         this.$store.state.fullscreen = true;
+        document.fullscreenElement.classList = "scrollbarBeGone";
```

Comment:
It's probably not a good idea to override the classes on the element. It'd be an easy place for new bugs to occur.
```suggestion
        document.querySelector("html").classList.add("scrollbarBeGone");
```